### DISCUSSION TITLE
сайт archlinux.org не открывается

### DISCUSSION THREAD

**Discussion by ghost** [bol-van/zapret#904] [Announcements]
не знаю в чем прикол, но некоторые сайты просто тупо не открываются, использую днскрипт с google
кроме сайта арча, неоткрывается офф сайт winrar.

при использовании blockcheck.sh
везде только это:

[attempt 1] curl: (28) Connection timed out after 2002 milliseconds
[attempt 2] curl: (28) Connection timed out after 2001 milliseconds
[attempt 3] curl: (28) Connection timed out after 2001 milliseconds
[attempt 4] curl: (28) Connection timed out after 2001 milliseconds
[attempt 5] curl: (28) Connection timed out after 2001 milliseconds

Раньше ютуб нормально работал,а сейчас плохо чето стал, может надо будет изменить настройки nfqws, те которые по дефолту шли все нормально было, поставил другие, перестало работать(

NFQWS_PORTS_TCP=80,443
NFQWS_PORTS_UDP=443
NFQWS_TCP_PKT_OUT=9
NFQWS_TCP_PKT_IN=3
NFQWS_UDP_PKT_OUT=9
NFQWS_UDP_PKT_IN=0
NFQWS_PORTS_TCP_KEEPALIVE=
NFQWS_PORTS_UDP_KEEPALIVE=
NFQWS_OPT="
--dpi-desync=fake,split2 --dpi-desync-ttl=3, --dpi-desync=fake --dpi-desync-repeats=5 <HOSTLIST_NOAUTO>"

можете скинуть свои варианты nfqws конфигурации.

Провайдер дом.ру

**Comment by Fenrir-0xFF** [comment]
Добавь --dpi-desync-fooling=md5sig,badsum или --dpi-desync-fooling=md5sig,badseq

**Reply by ghost** [reply]
с --dpi-desync-fooling=md5sig,badsum не сработало, --dpi-desync-fooling=md5sig,badseq тоже не работает.
тоже самое что и раньше: [attempt 1] curl: (28) Connection timed out after 2002 milliseconds
[attempt 2] curl: (28) Connection timed out after 2002 milliseconds
[attempt 3] curl: (28) Connection timed out after 2002 milliseconds
[attempt 4] curl: (28) Connection timed out after 2002 milliseconds
[attempt 5] curl: (28) Connection timed out after 2002 milliseconds

**Reply by ghost** [reply]
стоит теперь вот так:
NFQWS_OPT="
--dpi-desync=fake,split2 --dpi-desync-ttl=3, --dpi-desync=fake --dpi-desync-repeats=5  --dpi-desync-fooling=md5sig,badseq <HOSTLIST_NOAUTO>"

**Comment by ghost** [comment]


**Reply by bol-van** [reply]
почему так происходит написано в тексте при создании иссуе
там не место вопросам типа этого
в дискуссиях закрывается только неадекват

**Reply by ghost** [reply]
ладно

**Comment by ghost** [comment]
поставил дефолт значения nfqws такая же херня, может наспамить всеми возможными функциями?

**Reply by electrifying** [reply]
Запускай блокчек в несколько прогонов и ищи стратегию под своего провайдера

**Reply by ghost** [reply]
да уже пипец какой час ищу, щя попробую еще раз запустить

**Reply by ghost** [reply]
5 часов жду, дошел до вот этого:
- checking nfqws --dpi-desync=fake,multisplit --dpi-desync-fooling=badseq --wssize 1:6 --dpi-desync-split-pos=1 --dpi-desync-fake-tls=0x00000000
[attempt 1] curl: (28) Connection timed out after 2002 milliseconds
[attempt 2] curl: (28) Connection timed out after 2002 milliseconds
[attempt 3] curl: (28) Connection timed out after 2002 milliseconds
UNAVAILABLE code=28

**Reply by ghost** [reply]
@electrifying Может быть из за днскрипт такое? У меня стоит гугл

**Comment by bol-van** [comment]
Когда абсолютно ничего не работает из стратегий, первым делом стоит проверить есть ли конект на порт.
Если да, то нет ли блока по IP. Любые SNI вызывают зависание.
Это делает блокчек, но интерпретация результата лежит на юзере.

**Reply by ghost** [reply]
Окей, спасибо. Но у меня уже само по себе заработало лол. Я ниче не делал и сегодня сайт открылся, что с провайдерами не так...

**Reply by bol-van** [reply]
может глючить сам сайт или мрашрут до него с прова

**Reply by ghost** [reply]
сам сайт? У меня неработало больше 3 сайтов,а сейчас заработали. Ну я посмотрю что буду делать если снова открыватся не будет.

**Reply by ghost** [reply]
фак, перезапустил комп и опять нихрена не работает. Опять на значке вайфая вопросик появился...

**Reply by ghost** [reply]
конект на порта вроде нет кстати

**Reply by ghost** [reply]
ncat вот че дает: * port block tests ipv4 archlinux.org:80
ncat -z -w 2 95.217.163.246 80
95.217.163.246 does not connect. netcat code 1

**Reply by ghost** [reply]
Мне так кажется что из за днскрипт, у меня стоит google, opendns, yandex, cloudfare, наверное мне не надо было столько писать в конфиге, поэтому буду по одному оставлять, сначала гугл попробую, потом яндекс. Может в resolv.conf надо будет что нибудь поменять хз. Я уже менял когда настраивал. Причем до днскрипт и запрета сайт работал! Но еще! Что я заметил, у меня не работают сайты и на другом компе! (Да у меня два компа). Причем будто вся сеть зависит от моего компьютера. Это странно.

**Reply by ghost** [reply]
Роутер стоит на другом пк, и не зависит от моего.

**Reply by ghost** [reply]
ладно думаю что днскрипт не виноват.

**Reply by ghost** [reply]
причем я отключал службы запрета и днскрипт, все равно такая хрень

**Reply by ghost** [reply]
видимо что-то более глубокое чем просто запрет и днскрипт

**Reply by bol-van** [reply]
> ncat -z -w 2 95.217.163.246 80
> 95.217.163.246 does not connect. netcat code 1

это делает бессмысленными любые проверки
можно сразу останавливать

**Reply by ghost** [reply]
> > ncat -z -w 2 95.217.163.246 80
> > 95.217.163.246 does not connect. netcat code 1
> 
> это делает бессмысленными любые проверки можно сразу останавливать

и чё делать? Менять днс? Менять провайдера? Много вопросов но мало ответов.

**Reply by ghost** [reply]
> > ncat -z -w 2 95.217.163.246 80
> > 95.217.163.246 does not connect. netcat code 1
> 
> это делает бессмысленными любые проверки можно сразу останавливать

хей, я не знаю что случилось, но вот:
* port block tests ipv4 archlinux.org:80
ncat -z -w 2 95.217.163.246 80
95.217.163.246 connects
может из за того что я отключил QUIC, хотя везде все тоже самое: curl: (28) Operation timed out after 2002 milliseconds with 0 bytes received
UNAVAILABLE code=28
но хоть что-то

**Reply by ghost** [reply]
Я проверил через Тор. Сайт работает, только у меня такая фигня.

**Reply by bol-van** [reply]
это может быть не блок, а плохая связь на маршруте

**Reply by ghost** [reply]
> это может быть не блок, а плохая связь на маршруте

я понял что это не блок.
